src/api/admin.js

```diff
@@ -1,12 +1,66 @@
 import axios from '../axios';
 
 // authorization
+
+const createAdmin = (first_name, last_name, email, password, username) => {
```

Comment:
change the function to have one param called `body` ex: `createAdmin = (body) => {}`